README.md

```diff
@@ -1,4 +1,4 @@
-TangeloHub [![Build Status](https://travis-ci.org/Kitware/tangelohub.png?branch=master)](https://travis-ci.org/Kitware/tangelohub)
+TangeloHub [![Build Status](https://travis-ci.org/Kitware/flow.png?branch=master)](https://travis-ci.org/Kitware/flow)
```

Comment:
Yeah everything should be changed, but I'm ok with stopping at this for this PR. @zachmullen is also working on the switch-over from the Arbor side so I want to make sure you are in coordination with him in case he has a PR coming that will conflict with this.